### ISSUE TITLE
How can I upgrade background removed image quality?

### ISSUE THREAD

**Issue by shin0343** [alihassan143/backgroundremover#3]
This library is awesome! Thank you.

I'd like to improve background remover's quality as I like this.
Can I help improve an algorithm of library if you can guide?

**Comment by alihassan143** [comment]
what do you want to increase in the image?

**Comment by alihassan143** [comment]
if you are referring to detect every objects and remove background from it than currently this library only supports humans and animals picture background removing

**Comment by shin0343** [comment]
Yes, I understand that it is only possible to remove the background of photos of people and animals. What I wanted to do was to solve the case where in some cases the image quality of the resulting photo after removing the background deteriorates drastically.

**Comment by alihassan143** [comment]
> Yes, I understand that it is only possible to remove the background of photos of people and animals. What I wanted to do was to solve the case where in some cases the image quality of the resulting photo after removing the background deteriorates drastically.

i think you can do using image library
